portfolio/src/main/webapp/bandGen.js

```diff
@@ -0,0 +1,22 @@
+function getMood() {
+    let menu = document.getElementById("mood");
+    let mood = menu.options[menu.selectedIndex].text;
+    return mood;
+}
+
+function generateBand() {
+    let mood = getMood();
+    let band = addBands();
+    let bandContainer = document.getElementById("bandContainer");
+    let rand = Math.floor(Math.random() * Math.floor(2));
+    bandContainer.innerHTML = band[mood][rand];
+}
+
+function addBands(){
```

Comment:
nit: whitespace before `{`.

Also, the name of the function gives the impression you are adding the bands somewhere. Instead, you are creating and returning the bands name. Probably a better name would be `getBands` or `createBands`